tests/user.test.js

```
@@ -25,7 +25,7 @@ describe('User', () => {
     try {
       await sequelize.sync({ force: true });
     } catch (err) {
-      console.log(err);
+      // Error occurred
```

The `beforeAll` catch block swallows errors which hides setup failures. Instead of an empty catch, log the error and rethrow so test setup issues (DB connection, sync failures) are visible. This catch is here in the tests file; make it show the error (or remove the try/catch). See the test `beforeAll` block.